lib/nine_one_one/slack_service.rb

```diff
@@ -18,10 +21,15 @@ def notify(message)
 
     private
 
-    attr_reader :http, :path
+    attr_reader :channel, :http, :path, :username
 
     def request_body(message)
-      { text: message }.to_json
+      body = { text: message }
+
+      body[:channel]  = channel  unless channel.nil?
+      body[:username] = username unless username.nil?
```

Comment:
oh, ignore my comment, just re-read the configuration and noticed `String || nil?` :) my mistake ;)